tests/baselines/reference/objectSpreadStrictNull.errors.txt

```diff
@@ -76,10 +67,13 @@ tests/cases/conformance/types/spread/objectSpreadStrictNull.ts(42,5): error TS23
         fields = { ...fields, ...partialFields };
         // error: not optional, undefined remains
         fields = { ...fields, ...nearlyPartialFields };
-        ~~~~~~
-!!! error TS2322: Type '{ foo: number | undefined; bar: string | undefined; }' is not assignable to type 'Fields'.
-!!! error TS2322:   Types of property 'foo' are incompatible.
-!!! error TS2322:     Type 'number | undefined' is not assignable to type 'number'.
-!!! error TS2322:       Type 'undefined' is not assignable to type 'number'.
+                              ~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS2322: Type 'number | undefined' is not assignable to type 'number'.
+!!! error TS2322:   Type 'undefined' is not assignable to type 'number'.
+!!! related TS6500 tests/cases/conformance/types/spread/objectSpreadStrictNull.ts:31:5: The expected type comes from property 'foo' which is declared here on type 'Fields'
+                              ~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS2322: Type 'string | undefined' is not assignable to type 'string'.
+!!! error TS2322:   Type 'undefined' is not assignable to type 'string'.
+!!! related TS6500 tests/cases/conformance/types/spread/objectSpreadStrictNull.ts:32:5: The expected type comes from property 'bar' which is declared here on type 'Fields'

```

Comment:
now we end up reporting 2 errors (for two different properties) for a single spread element - is this OK or should I limit this to a single elementwise error per location?